fhir-cql/src/main/java/com/ibm/fhir/cql/helpers/ModelHelper.java

```diff
@@ -0,0 +1,506 @@
+/*
+ * (C) Copyright IBM Corp. 2021
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package com.ibm.fhir.cql.helpers;
+
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.time.ZoneOffset;
+import java.util.Optional;
+import java.util.UUID;
+
+import org.opencds.cqf.cql.engine.runtime.Code;
+
+import com.ibm.fhir.cql.Constants;
+import com.ibm.fhir.model.resource.Bundle;
+import com.ibm.fhir.model.resource.Library;
+import com.ibm.fhir.model.resource.Measure;
+import com.ibm.fhir.model.resource.Resource;
+import com.ibm.fhir.model.resource.ValueSet;
+import com.ibm.fhir.model.type.Attachment;
+import com.ibm.fhir.model.type.Canonical;
+import com.ibm.fhir.model.type.CodeableConcept;
+import com.ibm.fhir.model.type.Coding;
+import com.ibm.fhir.model.type.DateTime;
+import com.ibm.fhir.model.type.Element;
+import com.ibm.fhir.model.type.Extension;
+import com.ibm.fhir.model.type.Reference;
+import com.ibm.fhir.model.type.RelatedArtifact;
+import com.ibm.fhir.model.type.UnsignedInt;
+import com.ibm.fhir.model.type.Uri;
+import com.ibm.fhir.model.type.code.BundleType;
+import com.ibm.fhir.model.type.code.PublicationStatus;
+import com.ibm.fhir.model.type.code.RelatedArtifactType;
+
+/**
+ * Utilities for working with FHIR model objects.
+ */
+public class ModelHelper {

```

Comment:
from chatting with Lee/John/you we'll take a todo to incorporate this in the fhir-model